organizational/conduct.rst

```diff
@@ -0,0 +1,88 @@
+Code of Conduct
+===============
+
+Hacker Society [#hacsoc]_ is, as Tim wrote in :doc:`hacsoc`, about an open
+society:
```

Comment:
I would change this to restate the mission of Hacker Society.

# The Mission of Hacker Society

A time and place for people to learn about technology from each other, in order to build a community, to empower the individual, promote equality, and build an open society.

**Open Society**: An open society promotes freedom of thought and expression, personal responsibility, individualism, cooperation, and equality of all citizens.